.tool-versions

```diff
@@ -0,0 +1,6 @@
+rust 1.64
+make 4.3
+cmake 3.24.1
```

Comment:
do we need `cmake` and `protoc` even if when we are not using substrate?

Reply:
cmake maybe , protoc no, unless any dependency we may have needs it